huffman/test/table_from_symbol_bitsize_test.cpp

```diff
@@ -0,0 +1,69 @@
+#include "huffman/huffman.hpp"
+
+#include <boost/ut.hpp>
+
+#include <algorithm>
+#include <cstdint>
+#include <utility>
+#include <vector>
+
+auto main() -> int
+{
+  using ::boost::ut::expect;
+  using ::boost::ut::test;
+
+  namespace huffman = ::starflate::huffman;
+  using namespace huffman::literals;
+
+  test("table with DEFLATE canonical code, example 1") = [] {
```

Comment:
If these examples are from the RFC, please add that in a comment or put that in the test desc string (like `RFC example 1` instead of `example 1`).
Also, do we get any more code coverage from having 2 examples vs just 1? If not then probably just 1 is enough.

Reply:
I think coverage will be the same with one example vs two. I'll remove the smaller one and update the test description to include "RFC"

Reply:
Updated descriptions and added links to RFC examples. Coverage comes from the second test. First test verifies usability in constant expressions. I've replaced `expect` with `static_assert` to make that more clear.